packages/lib/convertMinimumBookingNoticeToMinutes.ts

```diff
@@ -0,0 +1,10 @@
+export default function convertMinimumBookingNoticeToMinutes(type: string, minNotice: number) {
+  if (type == "minute") {
+    return minNotice;
+  } else if (type == "hour") {
+    return minNotice * 60;
+  } else if (type == "day") {
+    return minNotice * 1440;
```

Comment:
```suggestion
  if (type === "minute") {
    return minNotice;
  } else if (type === "hour") {
    return minNotice * 60;
  } else if (type === "day") {
    return minNotice * 1440;
```

Reply:
Thank you so much for this, I need to remember to use `===` instead of `==` more often!